include/wx/qt/textctrl.h

```diff
@@ -73,7 +73,8 @@ class WXDLLIMPEXP_CORE wxTextCtrl : public wxTextCtrlBase
 private:
     QLineEdit *m_qtLineEdit;
     QTextEdit *m_qtTextEdit;
-
+    virtual QLineEdit *GetEditable() const wxOVERRIDE;
```

Comment:
All the other wx styles declare the functions _before_ the member variables, not after. Could you please do it like this here too for consistency?